### PR TITLE
ADD documentation for some missing CLI options

### PR DESCRIPTION
**Pull request by fgalan**
@kzangeli @crbrox

### CODE REVIEW

File: doc/manuals/admin/cli.md [hunk 1]
```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
     Active-Active Orion configuration "none" is recommended.
 -   **-mutexTimeStat**. Include semaphore waiting time in the
     "/statistics" information. It may have performance impact.
--   **-subCacheIval**. Interval in seconds between calls to Subscription Cache refresh.
+-   **-subCacheIval**. Interval in seconds between calls to subscription cache refresh. Zero
+    value means "no refresh" (default value and the recomnended one for mono-CB
+    deployments).
+-   **-noCache**. Disables context subscription cache, so subscriptions searchs are
+    always done at DB (not recommneded but useful for debugging).
+-   **-notificationMode** *(Experimental option)*. Allows to select notification mode, either:
```
Comment (fgalan) [inline doc/manuals/admin/cli.md]: I foresee potential changes in the syntax of this CLI option (e.g. to specify pipeling, pool size, etc.) so better to mark it as experimental by the moment in the documentation.

NTC (just informative).

File: doc/manuals/admin/cli.md [hunk 2]
```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
     Active-Active Orion configuration "none" is recommended.
 -   **-mutexTimeStat**. Include semaphore waiting time in the
     "/statistics" information. It may have performance impact.
--   **-subCacheIval**. Interval in seconds between calls to Subscription Cache refresh.
+-   **-subCacheIval**. Interval in seconds between calls to subscription cache refresh. Zero
+    value means "no refresh" (default value and the recomnended one for mono-CB
+    deployments).
+-   **-noCache**. Disables context subscription cache, so subscriptions searchs are
```
Comment (kzangeli) [inline doc/manuals/admin/cli.md]: Typo: searches

File: CHANGES_NEXT_RELEASE [hunk 3]
```diff
@@ -26,4 +26,4 @@ Fix:  Add charset=utf-8 for notifications (Issue #1340)
 Add:  Functional test to PUT/POST/PATCH with 'empty' JSON object as payload (Issue #1305)
 Fix:  performance problem in NGSIv2 API due to uncontrolled log trace printing
 Add:  empty attribute values supported in NGSIv1 API operations (Issue #1358)
-Add:  Add CLI argument for notification mode (Issue #1370)
+Add:  Add CLI argument for notification mode. Default mode is 'transient' (previous version used 'permanent' implicitely) (Issue #1370)
```
Comment (kzangeli) [inline CHANGES_NEXT_RELEASE]: Added
Implicitly

File: doc/manuals/admin/cli.md [hunk 4]
```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
     Active-Active Orion configuration "none" is recommended.
 -   **-mutexTimeStat**. Include semaphore waiting time in the
     "/statistics" information. It may have performance impact.
--   **-subCacheIval**. Interval in seconds between calls to Subscription Cache refresh.
+-   **-subCacheIval**. Interval in seconds between calls to subscription cache refresh. Zero
+    value means "no refresh" (default value and the recomnended one for mono-CB
```
Comment (kzangeli) [inline doc/manuals/admin/cli.md]: A value of zero means ...

File: doc/manuals/admin/cli.md [hunk 5]
```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
...
+    deployments).
+-   **-noCache**. Disables context subscription cache, so subscriptions searchs are
+    always done at DB (not recommneded but useful for debugging).
```
Comment (kzangeli) [inline doc/manuals/admin/cli.md]: recommended

File: doc/manuals/admin/cli.md [hunk 6]
```diff
@@ -121,4 +121,19 @@ The list of available options is the following:
...
+-   **-notificationMode** *(Experimental option)*. Allows to select notification mode, either:
+    `transient`, `permanent` or `none`. Default mode is `transient`.
+    * In transient mode, connections are closed by CB just after sending the notification.
+    * In permanent connection mode, a permanent connection is created the first time a notification
+      is sent to a given URL path (if the receiver support permanent connections). Next notifications to the same
```
Comment (kzangeli) [inline doc/manuals/admin/cli.md]: Next -> Following

File: doc/manuals/admin/cli.md (hunk 4 above)
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Fixed in 309bea3

File: doc/manuals/admin/cli.md (hunk 2 above)
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Fixed in 309bea3

File: doc/manuals/admin/cli.md (hunk 5 above)
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Fixed in 309bea3

File: doc/manuals/admin/cli.md (hunk 6 above)
Comment (fgalan) [inline doc/manuals/admin/cli.md]: Fixed in 309bea3

**Comment by kzangeli** [comment]
LGTM
